book/src/external-snapshotter.md

```diff
@@ -23,6 +23,31 @@ To use the snapshot beta feature, a snapshot controller is also required. For mo
 
 In the Beta version, the snapshot controller will be watching the Kubernetes API server for `VolumeSnapshot` and `VolumeSnapshotContent` CRD objects. The CSI `external-snapshotter` sidecar only watches the Kubernetes API server for `VolumeSnapshotContent` CRD objects. The CSI `external-snapshotter` sidecar is also responsible for calling the CSI RPCs CreateSnapshot, DeleteSnapshot, and ListSnapshots.
 
+#### VolumeSnapshotClass Parameters
+
+When provisioning a new volume snapshot, the CSI `external-snapshotter` sets the `map<string, string> parameters` field in the CSI `CreateSnapshotRequest` call to the key/values specified in the `VolumeSnapshotClass` it is handling.
+
+The CSI `external-snapshotter` (v2.1.0+) also reserves the parameter keys prefixed with `csi.storage.k8s.io/`. Any `VolumeSnapshotClass` keys prefixed with `csi.storage.k8s.io/` are not passed to the CSI driver as an opaque `parameter`.
```

Comment:
Some keys were added in v1.0.1 and some were added in v2.1.0.

Reply:
Would you prefer I remove the parentheses on line 30 and then put the versions next to the corresponding parameter keys below?

Reply:
Yes, that sounds good.